### ISSUE TITLE
Issues with current version of react-native

### ISSUE THREAD

**Issue by malonehedges** [akveo/react-native-reddit-reader#1]
Initially `react-native run-ios` was unrecognized. I updated the react-native version as suggested in this [StackOverflow answer](https://stackoverflow.com/questions/36119358/react-native-command-run-ios-unrecognized). Now it will fail to build with the following error:

```
** BUILD FAILED **


The following commands produced analyzer issues:
	Analyze Modules/RCTRedBox.m

(1 command with analyzer issues)

The following build commands failed:
	Ld build/Build/Products/Debug-iphonesimulator/ReactNativeRedditReader.app/ReactNativeRedditReader normal x86_64
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/ReactNativeRedditReader.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=22):
Failed to install the requested application
The bundle identifier of the application could not be determined.
Ensure that the application's Info.plist contains a value for CFBundleIdentifier.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/ReactNativeRedditReader.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

cc @smartapant

**Comment by smartapant** [comment]
@malonehedges quite an old version of RN was used in this app, so updating it to the newest one is really painful

**Comment by malonehedges** [comment]
@smartapant after I left the issue I actually went and updated the app to the most recent react-native version and ES6 syntax haha

I've published it under the MIT License at [malonehedges/RedditReader](https://github.com/malonehedges/RedditReader).

I didn't submit a PR because I started from scratch and changed a pretty significant amount of the coding style. Let me know if you'd be interested in merging somehow!
